Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build stage
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Copy the application code into the container
+COPY . .
+
+# Runtime stage
+FROM python:${PYTHON_VERSION}-slim
+WORKDIR /app
+
+# Copy the entire /app directory from the build stage
+COPY --from=base /app /app
+
+# Install dependencies and run migrations
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt && \
+    python manage.py migrate
+
+# Expose port 8080 to the host
+EXPOSE 8080
+
+# Define the command to run the application
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
Please add EOF(End-Of-File), need to add one empty line 
![image](https://github.com/mate-academy/devops_todolist/assets/87720583/067c5956-dff1-4b06-a624-d9b044a729b1)